.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,4 @@
+1. [Read and sign the CLA](https://cla.js.foundation/webpack/webpack.js.org). This needs to be done only once. PRs that haven't signed it won't be accepted.
+2. Make sure your PR complies with [the writer's guide](https://webpack.js.org/writers-guide/).
```

Comment:
Yep, but we don’t have any dev docs atm, I will update them as soon as we got them 😛